tests/Credentials/UserRefreshCredentialsTest.php

```diff
@@ -110,6 +110,27 @@ public function testInitalizeFromAFile()
             new UserRefreshCredentials('scope/1', $keyFile)
         );
     }
+
+    /**
+     * @expectedException \PHPUnit\Framework\Error\Warning
```

Comment:
If I remember correctly depending on the version of PHPUnit installed this could not exist. There wasn't a class added for forwards compatibility like `TestCase`.